Gruntfile.js

```diff
@@ -146,7 +146,9 @@ module.exports = function(grunt) {
         files: {
           // Destination: [source files]
           'out/app_engine/js/apprtc.debug.js': [
-	    'src/web_app/js/analytics.js',
+            'node_modules/jssha/src/sha.js',
```

Comment:
Should we consider serving these from the same place as where callstats is being served from (e.g. static content)?